readme.md

```diff
@@ -149,6 +155,13 @@ Execute a command synchronously through the system shell.
 
 Returns the same result object as [`child_process.spawnSync`](https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options).
 
+### spawned.cancel()
+
+Cancel a process spawned using execa. Calling this method kills it.
+
+Throws an error with `error.canceled` equal to `true`, provided that the process gets canceled.
+Process would not get canceled if it has already exited or has been killed by `spawned.kill()`.
```

Comment:
I think you can just stop at `already exited` since if it has been killed, then it has exited.